ShiftLogger.Bata13/STUDY.ASP.ShiftLoggerTryThree/Data/DataContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore;
+using STUDY.ASP.ShiftLoggerTryThree.Models;
+
+namespace STUDY.ASP.ShiftLoggerTryThree.Data
+{
```

Comment:
Next time try to use semicolon after defined namespace. This way you provide file scoped namespace, which will save 2 lines of code and 1 line of indentation which increases readability 😁

`namespace STUDY.ASP.ShiftLoggerTryThree.Data;`